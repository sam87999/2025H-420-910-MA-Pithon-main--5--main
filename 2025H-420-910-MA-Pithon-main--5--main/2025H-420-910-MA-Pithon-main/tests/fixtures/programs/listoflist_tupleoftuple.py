# Test de Alexandrine Dub√©
print([[1,2],[2,4],[6,8]] + [1,2,3,4,4])
print([["oki"],[2,4],[6,8]])
print([1,2] in [[1,2],[2,4],[6,8]])
print(((1,2,3),(2,3,4),(4,56,5)) + ("toto"+"wow","oki"))
print(((1,2,"oki"), ("wow", 83, True)))
print(("oki","wow") in ((1,2,3),(2,3,4),("oki","wow"),(4,56,5)))