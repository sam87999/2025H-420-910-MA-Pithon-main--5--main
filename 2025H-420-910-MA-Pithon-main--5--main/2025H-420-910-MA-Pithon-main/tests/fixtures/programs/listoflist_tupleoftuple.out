[[1, 2], [2, 4], [6, 8], 1, 2, 3, 4, 4]
[['oki'], [2, 4], [6, 8]]
True
((1, 2, 3), (2, 3, 4), (4, 56, 5), 'totowow', 'oki')
((1, 2, 'oki'), ('wow', 83, True))
True
